Actions/Sign/action.yaml

```diff
@@ -0,0 +1,77 @@
+name: PowerShell script
+author: Microsoft Corporation
+inputs:
+  shell:
+    description: Shell in which you want to run the action (powershell or pwsh)
+    required: false
+    default: powershell
+  azureKeyVaultURI:
+    description: URI of the Azure Key Vault
+    required: true
+  azureKeyVaultClientID:
+    description: The Client ID of the service principal used to access the Azure Key Vault
+    required: true
+  azureKeyVaultClientSecret:
+    description: The Client Secret of the service principal used to access the Azure Key Vault
+    required: true
+  azureKeyVaultTenantID:
+    description: The Tenant ID of the service principal used to access the Azure Key Vault
+    required: true
+  azureKeyVaultCertificateName:
+    description: The name of the certificate in the Azure Key Vault
+    required: true
+  pathToFiles:
+    description: The path to the files to be signed
+    required: true
+  fileExtensionsToSign:
+    description: The file extensions to be signed
+    required: false
+    default: .app
+  timestampService:
+    description: The URI of the timestamp server
+    required: false
+    default: http://timestamp.digicert.com
+  digestAlgorithm:
+    description: The digest algorithm to use for signing and timestamping
+    required: false
+    default: SHA256
+runs:
+  using: composite
+  steps:
+    - name: Install Azure SignTool
+      shell: ${{ inputs.shell }}
+      run: dotnet tool install --global AzureSignTool --version 4.0.1
+
+    - name: run
+      shell: ${{ inputs.shell }}
+      id: Sign
+      env:
+        _azureKeyVaultURI: ${{ inputs.azureKeyVaultURI }}
```

Comment:
This isn't really needed.

From the docs:

> When you specify an input in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name `INPUT_<VARIABLE_NAME>`.

 https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
 
 Clarification: I'm not suggesting you use `INPUT_<VARIABLE_NAME>`, I suggest you use, for example, `${{ inputs.azureKeyVaultURI }}` instead of `_azureKeyVaultURI`